tests/integration/test_dummy_integration.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+from helpers import PROTEUS_ROOT, resize_to_match
+from matplotlib.testing.compare import compare_images
+from pandas.testing import assert_frame_equal
+
+from proteus import Proteus
+from proteus.utils.coupler import ReadHelpfileFromCSV
+
+IMAGE_LIST = (
+        "plot_elements.png",
+        "plot_escape.png",
+        "plot_fluxes_global.png",
+        "plot_global_lin.png",
+        "plot_global_log.png",
+        "plot_observables.png",
+        "plot_sflux.png",
+        )
+
+@pytest.fixture
+def test_dummy_run():
+    config_path = PROTEUS_ROOT / 'input' / 'dummy.toml'
+    ref_output = PROTEUS_ROOT / 'tests' / 'data' / 'integration' / 'dummy'
+
+    runner = Proteus(config_path=config_path)
+
+    runner.start()
+
+    output = Path(runner.directories['output'])
+
+    hf_all = ReadHelpfileFromCSV(output)
+    hf_all_ref = ReadHelpfileFromCSV(ref_output)
+
+    assert_frame_equal(hf_all, hf_all_ref, rtol=1e-2)
+
+def test_plot_dummy_integration(test_dummy_run):
+
+    out_dir = PROTEUS_ROOT / 'output' / 'dummy'
+    ref_dir = PROTEUS_ROOT / 'tests' / 'data' / 'integration' / 'dummy'
+
+    for image in IMAGE_LIST:
+        out_img = out_dir / image
+        ref_img = ref_dir / image
+        tolerance = 14
```

Comment:
If we need to make the tolerance this high, then comparing the plots has little use because we will not catch actual differences.

```suggestion
        tolerance = 14
```

Reply:
Will create a separate issue to solve this.